app/controllers/grade_entry_forms_controller.rb

```diff
@@ -264,13 +267,16 @@ def get_mark_columns
           id: 'total_marks',
           content: t('grade_entry_forms.grades.total') \
                    + ' ' + grade_entry_form.out_of_total.to_s,
+          sortable: true,
+          compare: "compare_gradebox"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.